src/app/backend/resource/deployment/util/deploymentutil.go

```diff
@@ -0,0 +1,100 @@
+package util
```

Comment:
Just tried to reuse them and unfortunately the are using different extensions package version. We have to leave it as is for now.